src/components/bottom_bar/bottom_bar.tsx

```diff
@@ -132,9 +132,9 @@ export const OuiBottomBar = forwardRef<
       bodyClassName,
       landmarkHeading,
       usePortal = true,
-      left = 0,
-      right = 0,
-      bottom = 0,
+      left,
+      right,
+      bottom,
```

Comment:
hmm... So the properties are no longer _always_ set inline, but if these props are used the styles will be. That seems reasonable, and solves are problem in OpenSearch Dashboards. But I wonder if we should consiter getting rid of these props altogether.

Reply:
I can remove them, but I believe that these properties provide us with more opportunities for customization and control over the `OuiBottomBar`. Additionally, removing them altogether could potentially limit our options in the future

Reply:
Yeah i too like the idea of removing them, but that will be a breaking change since it changes the api of the bottom bar

Reply:
Lets create a follow-up issue for this and create a separate PR so that these changes can be backported